server/routes/summary_routes.ts

```diff
@@ -17,6 +17,35 @@ const LOG_PATTERN_SUMMARY_AGENT_CONFIG_ID = 'os_summary_with_log_pattern';
 const OS_INSIGHT_AGENT_CONFIG_ID = 'os_insight';
 const DATA2SUMMARY_AGENT_CONFIG_ID = 'os_data2summary';
 
+export function postprocessing(output: string) {
```

Comment:
Let's follow camel-case convention
```suggestion
export function postProcessing(output: string) {
```

Reply:
ok